docs/api.rst

```diff
@@ -15,6 +15,14 @@ The main class for controlling GPIO (parallel) connected LCDs.
 
 .. autoclass:: RPLCD.gpio.CharLCD
 
+CharLCD (lgpio)
+===============
+
+The main class for controlling GPIO (parallel) connected LCDs on Raspberry Pi 5
```

Comment:
This should probably be 5+, right? (Assuming that the GPIO won't change again in the next version.)

```suggestion
The main class for controlling GPIO (parallel) connected LCDs on Raspberry Pi 5+
```